### DISCUSSION TITLE
CoolWallet Coin Tester 畫面空白

### DISCUSSION THREAD

**Discussion by fantersee** [CoolBitX-Technology/coolwallet-js-sdk#250] [Q&A]
- Description
安裝 CoolWallet Coin Tester 至
 ....
 cd ../coin-tester
 npm ci
 npm run dev 
 以後 網頁呈現空白

- Expected Behavior
出現測試畫面

- Screenshots or appendix

![截圖 2022-01-18 下午12 15 24](https://user-images.githubusercontent.com/4178903/149873664-5a2387ee-dce8-45b5-a8e3-9b4c53939769.png)

**Comment by hank121314** [comment]
Could you try to pull the latest changes in the master branch?
This issue should be fixed.

If the issue still exists, please provide the console log of browser.

Thanks!
